src/app.js

```
@@ -1 +1,134 @@
-// write code here
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('node:fs');
+const path = require('node:path');
+
+const commands = process.argv.slice(2);
+const [filePath, destinationPath] = commands;
+let currDir = null;
+let destDir = null;
+let currFile = null;
+let destFile = null;
+
+if (typeof filePath === 'string') {
+  currDir = path.dirname(filePath);
+  currFile = path.basename(filePath);
+}
+
+if (typeof destinationPath === 'string') {
+  destDir = path.dirname(destinationPath);
+
+  destFile = path.basename(destinationPath) || currFile;
+}
+
+if (conditionsCheck()) {
+  moveFile(filePath, destinationPath);
+}
+
+function conditionsCheck() {
+  if (commands.length < 2) {
+    console.error(`Check arguments in comand line`);
+
+    return false;
+  } else if (isDirectory(filePath)) {
+    console.error(`Source is a directory!`);
+
+    return false;
+  } else if (!fs.existsSync(destDir) && !destinationPath.endsWith(path.sep)) {
+    console.error(`Destination directory does not exist!`);
+
+    return false;
```

According to the task description, in case the destination directory does not exist, an **error must be thrown**. Currently, you are only printing the error message to the console. You should throw an error instead of returning `false`.